test/OpenTelemetry.Tests/Trace/LinkTest.cs

```diff
@@ -34,7 +34,11 @@ public LinkTest()
             this.attributesMap.Add("MyAttributeKey1", 10L);
             this.attributesMap.Add("MyAttributeKey2", true);
             this.attributesMap.Add("MyAttributeKey3", 0.005);
-            this.tags = new ActivityTagsCollection(this.attributesMap);
+            this.tags = new SpanAttributes();
+            this.tags.Add("MyAttributeKey0", "MyStringAttribute");
+            this.tags.Add("MyAttributeKey1", 10L);
+            this.tags.Add("MyAttributeKey2", true);
+            this.tags.Add("MyAttributeKey3", 0.005);
```

Comment:
Sanity check: are arrays covered on `SpanAttributes` tests?

Reply:
yes, just a few but yes.

Reply:
i will add the other types to be complete! 👍